src/main/resources/public/js/src/projectEvents/projectEventsItemView.js

```diff
@@ -67,7 +67,11 @@ define(function (require) {
                         var $valueView;
                         var oldLinks = [];
                         var newLinks = [];
-                        if (this.view.model.get('actionType') === 'post_issue' || this.view.model.get('actionType') === 'attach_issue') {
+                        if (
+                            this.view.model.get('actionType') === 'post_issue'
```

Comment:
`this.view.model.get('actionType')` - I think it's time to make a variable for this

Reply:
Fixed